### DISCUSSION TITLE
Specifications for IoT Data collected over time and locations

### DISCUSSION THREAD

**Discussion by cOsprey** [radiantearth/stac-spec#1298] [General STAC Q&A]
Hi all,
I am not sure if this is the correct place to raise an issue, but I am trying to integrate STAC for IoT Sensor Data, and I am not getting any resources for it. I would like to know how to approach the problem. Many resources exist for Satellite Datasets but not for the sensor data.
Any suggestions or feedback would be really helpful. Thanks!

**Comment by m-mohr** [comment]
How do you plan to expose the sensor data? Let's say you have 100 temperature measurements for a single sensor somewhere in Germany. Would you want to expose all measurements individually or would you want to expose them all together as one file, e.g. CSV? For the first case you'd probably look at something else than STAC, for the second usecase you could use STAC.
